resources/styles/book-content/note.less

```diff
@@ -9,6 +9,11 @@
     display: inline-block;
     margin: @book-content-note-with-background-margin;
   }
+  // hide note title if data-label attribute is blank or missing
```

Comment:
why not move this up to wrap the `&:before` above so that it's like this:

```css
&[data-label]:not([data-label=""]) {
  &:before {
    blah: blahblah;
  }
}
```